builder/scaleway/config.go

```diff
@@ -263,6 +268,10 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 		c.ShutdownTimeout = "5m"
 	}
 
+	if c.SnapshotTimeout == 0 {
+		c.SnapshotTimeout = 5 * time.Minute
```

Comment:
Could you define this as a constant? Same for all hard-coded values of timeouts. Doing so will make it easier to find quickly all the default timeout